benchmark/kernels/fused_moe_triton/README.md

```diff
@@ -8,16 +8,24 @@ This directory contains benchmarking tools for MoE (Mixture of Experts) kernels.
 
 Example usage:
 ```bash
+# Tune Mixtral-8x7B with default settings
+python benchmark/kernels/fused_moe_triton/tuning_fused_moe_triton.py \
+    --model mistralai/Mixtral-8x7B-Instruct-v0.1 \
+    --tune
+
 # Tune Qwen2-57B with FP8 and TP=4
 python benchmark/kernels/fused_moe_triton/tuning_fused_moe_triton.py \
     --model Qwen/Qwen2-57B-A14B-Instruct \
     --tp-size 4 \
     --dtype fp8_w8a8 \
     --tune
 
-# Tune Mixtral-8x7B with default settings
+# Tune DeepSeek-V3 with FP8, TP=8 and n_share_experts_fusion=8
 python benchmark/kernels/fused_moe_triton/tuning_fused_moe_triton.py \
-    --model mistralai/Mixtral-8x7B-Instruct-v0.1 \
+    --model deepseek-ai/DeepSeek-V3-0324 \
+    --tp-size 8 \
+    --n-share-experts-fusion 8 \
+    --dtype fp8_w8a8 \
```

Comment:
nit: do we need this, or will the dtype be automatically derived?

Reply:
If for a MoE model, inference can be done using both fp8 and bf16 dtype, then inferring the dtype from the HF model's config is inappropriate. Manually specifying allows the user to clearly know which dtype they are tuning.